### ISSUE TITLE
Docker test image has no linux/arm64/v8 variant

### ISSUE THREAD

**Issue by lloeki** [dense-analysis/ale#4377]
The [images published](https://hub.docker.com/r/denseanalysis/ale/tags) have only `linux/amd64`, but it's much slower on an `aarch64` CPU (e.g a Mac with Apple Silicon M1 or M2):

```
Docker run image denseanalysis/ale:c782d193fd213b9829ab60bc4e0515ad ready
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

If one encounters the following instead:

```
Downloading run image denseanalysis/ale:c782d193fd213b9829ab60bc4e0515ad
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
exec /sbin/run_vim: exec format error
```

Then run that beforehand:

```
docker run --privileged --rm tonistiigi/binfmt --install amd64
```

**Comment by w0rp** [comment]
Is there any way we can fix this so it just works?

**Comment by lloeki** [comment]
Yes, by building and pushing for both arches simultaneously using docker buildx.

This is best achieved in CI, I've done that multiple times before. I can give some pointers, maybe even open a PR.

**Comment by w0rp** [comment]
> Yes, by building and pushing for both arches simultaneously using docker buildx.
> 
> This is best achieved in CI, I've done that multiple times before. I can give some pointers, maybe even open a PR.

If you can open a PR for it, I'll take it. I don't have ARM machines to test on myself.

**Comment by w0rp** [comment]
Should be fixed in #4421, which I just merged.
